7_react/react1/week2/todo-list-app/src/TodoItem.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const TodoItem = ({ id, description, deadline, done, toggleDone, deleteTask }) => {
+  return (
+    <div style={{ textDecoration: done ? 'line-through' : 'none' }}>
+      <input type="checkbox" checked={done} onChange={toggleDone} />
+      <span>{description} {deadline}</span>
+      <button onClick={deleteTask}>Delete</button>
```

Comment:
On the button the label comes from the value so this is good!
`<button> THIS WILL BE READ BY SCREEN READER </button>`